src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/client/ClientHelper.java

```diff
@@ -14,7 +15,7 @@ public final class ClientHelper {
      * @return The player logged into the current Minecraft instance.
      */
     @Nullable
-    public static Player getLocalPlayer() {
+    public static LocalPlayer getLocalPlayer() {
```

Comment:
This could cause class loading errors for client only methods in common classes that call this

Reply:
Fair enough, changed